### ISSUE TITLE
How to import 826 txt files from Simplenote?

### ISSUE THREAD

**Issue by frankfarm** [standardnotes/forum#1004]
I am attempting to migrate from Simplenote to Standard Notes. Following the instructions at https://standardnotes.org/help/30/how-can-i-import-my-notes-from-simplenote I successfully exported my notes and converted them using the plaintext converter tool. So far so good. Now when attempting to import to Standard Notes I find that both in the web app and in the macOS app I can select only one file at a time. I have 826 notes. How can I import them all at one time rather than 826 times?

**Comment by frankfarm** [comment]
I have found that Apple Notes easily imported all my notes in a single import, so I am using that for now. I am still interested in hearing your answer in case Apple Notes later doesn't suit my needs. Thanks.

**Comment by joshcsullivan** [comment]
You can try this: https://standardnotes.org/help/30/how-can-i-import-my-notes-from-simplenote ?

**Comment by frankfarm** [comment]
Thanks. The url you provided is the same one that I had used and which did not work. After some time, however, I realized that `sn-import-file.txt` was the filename I was supposed to have imported. The url at https://standardnotes.org/help/30/how-can-i-import-my-notes-from-simplenote mentions "a new, single file will be generated," but it does not mention its filename by name, so I was confused because `sn-import-file.txt` was included in the bundle along with hundreds of txt files and other folders and I did not notice it at first. I found this path a frustrating one, but it is now resolved, and I will try Standard Notes alongside Apple Notes and see which I like better. Thanks again.
